### ISSUE TITLE
SpeedIndicater doesn't show up in lockscreen.

### ISSUE THREAD

**Issue by yemii** [seasonyuu/Xposed-NetworkSpeedIndicator#1]
Thank you for creating such application. However, i would really love it if you can modify it to work in lock screen.
Regards

**Comment by seasonyuu** [comment]
Yep I'm trying to make it work in lockscreen too, when you set the position as left, you will see the traffic text in the notification icon area, but it doesn't work fine. I'm trying to look for a better way to make it display in lockscreen, please wait for my situation
